<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Farmacia Kaya</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44, 62, 80);</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  color: rgb(255, 255, 255);
  background-color: rgb(0, 103, 192);
}

QPushButton:hover{ 
	background-color: rgb(166, 202, 255);
}

QPushButton:pressed{
	background-color: rgb(116, 255, 121);
}</string>
     </property>
     <property name="text">
      <string>Histórico</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  color: rgb(255, 255, 255);
  background-color: rgb(0, 103, 192);
}

QPushButton:hover{ 
	background-color: rgb(166, 202, 255);
}

QPushButton:pressed{
	background-color: rgb(116, 255, 121);
}</string>
     </property>
     <property name="text">
      <string>Stock</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  color: rgb(255, 255, 255);
  background-color: rgb(0, 103, 192);
}

QPushButton:hover{ 
	background-color: rgb(166, 202, 255);
}

QPushButton:pressed{
	background-color: rgb(116, 255, 121);
}</string>
     </property>
     <property name="text">
      <string>Ajuda</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  color: rgb(255, 255, 255);
  background-color: rgb(0, 103, 192);
}

QPushButton:hover{ 
	background-color: rgb(255, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(116, 255, 121);
}</string>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/newPrefix/tia.jpg)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>1031</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>981</width>
       <height>401</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>122</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descrição</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pagamento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Endereço</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preço</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>490</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
font-size: 13px;
text-align: center;
border-radius: 10px;
	background-color: rgb(211, 211, 211);
}</string>
     </property>
     <property name="placeholderText">
      <string>ID do Pedido</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  padding: 5px 15px;
  font-size: 15px;
  text-align: center;
  color: #fff;  
 background-color: rgb(195, 172, 204);
  border-radius: 13px;

}

QPushButton:hover{
background-color: purple;
}

QPushButton:pressed{
  background-color: #3e8e41;

}</string>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>530</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 207, 207);</string>
     </property>
     <property name="currentText">
      <string extracomment="Estado do Pedido"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Enviado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Recebido</string>
      </property>
     </item>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>1041</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(57, 207, 182);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>-4</y>
        <width>161</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>11:50:40</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>91</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Aberto</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>1021</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Light">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Midlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="NoRole">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Light">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Midlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="NoRole">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Light">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Midlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>195</red>
              <green>172</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="NoRole">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>108</green>
              <blue>102</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
  padding: 5px 15px;
  font-size: 15px;
  text-align: center;
  color: #fff;  
	background-color: rgb(195, 172, 204);
  border-radius: 13px;

}

QPushButton:hover{
background-color: purple;
}

QPushButton:pressed{
  background-color: #3e8e41;

}</string>
        </property>
        <property name="text">
         <string>Novo Pedido</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="styleSheet">
         <string notr="true">QPushButton{
  padding: 5px 15px;
  font-size: 15px;
  text-align: center;
  color: #fff;  
	background-color: rgb(195, 172, 204);
  border-radius: 13px;

}

QPushButton:hover{
background-color: purple;
}

QPushButton:pressed{
  background-color: #3e8e41;

}</string>
        </property>
        <property name="text">
         <string>Receita</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
  padding: 5px 15px;
  font-size: 15px;
  text-align: center;
  color: #fff;  
	background-color: rgb(195, 172, 204);
  border-radius: 13px;

}

QPushButton:hover{
background-color: purple;
}

QPushButton:pressed{
  background-color: #3e8e41;

}</string>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>widget_2</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>comboBox</zorder>
    <zorder>frame</zorder>
    <zorder>tableWidget</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPedidos_2"/>
    <addaction name="actionSobre_3"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSobbre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionPedidos">
   <property name="text">
    <string>Pedidos</string>
   </property>
  </action>
  <action name="actionExtrato">
   <property name="text">
    <string>Extrato</string>
   </property>
  </action>
  <action name="actionPadr_o">
   <property name="text">
    <string>Padrão</string>
   </property>
  </action>
  <action name="actionLight_Mode">
   <property name="text">
    <string>Light Mode</string>
   </property>
  </action>
  <action name="actionMedicamentos">
   <property name="text">
    <string>Medicamentos</string>
   </property>
  </action>
  <action name="actionPedidos_2">
   <property name="text">
    <string>Exportar Pedidos</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionConfiguracoes">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
  <action name="actionAjuda">
   <property name="text">
    <string>Ajuda</string>
   </property>
  </action>
  <action name="actionSobre_2">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionConfigura_es">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
  <action name="actionSobre_3">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="kaya.qrc.txt"/>
 </resources>
 <connections/>
</ui>
